static/css/style.sass

```diff
@@ -0,0 +1,68 @@
+@import 'variables'
+@import 'header'
+@import 'form'
+@import 'country_card'
+@import 'detailed_country_card'
+
+a
+    color: inherit
+    text-decoration: none    
+a:visited
+    color: inherit 
+
+body
+    background-color: var(--background-color)
+    font-family: 'Nunito Sans'
+    font-size: 14px
+    font-weight: $lightWeight
+    color: var(--text-color)
+
+.country-card-container
+    display: flex
+    flex-direction: column
+    flex-wrap: wrap
+    padding: 30px 20px 0 20px
+    @media (min-width: 768px)
+        padding: 30px 60px 0 60px
+        display: grid
+        grid-template-columns: repeat(auto-fill, minmax(280px, 1fr)) 
+        column-gap: 80px
+    @media (min-width: 1120px)
+        padding: 50px 60px 0 60px    
+
+            
+.error-message
+    font-size: 20px
+    padding-top: 190px
+    margin-bottom: 190px
+    height: 500px
+    text-align: center
+    grid-column: 1 / -1
+
+.fa-circle-exclamation
+    color: red
+    font-size: 25px
+
+.loader
```

Comment:
Ik denk wat je moet doen is volgende.

-Maak de loader container `position:fixed`
-Zorg dat het hele scherm vult, en voeg een transparent/zwart achtergrond
-positioneer je icoontje in het midden  `.fa-spinner`
-maak hem wat groter